docopt.h

```diff
@@ -67,7 +43,8 @@ namespace docopt {
 	/// @throws DocoptExitHelp if 'help' is true and the user has passed the '--help' argument
 	/// @throws DocoptExitVersion if 'version' is true and the user has passed the '--version' argument
 	/// @throws DocoptArgumentError if the user's argv did not match the usage patterns
-	std::map<std::string, value> DOCOPT_API docopt_parse(std::string const& doc,
+	DOCOPT_API
```

Comment:
I'm a little nervous about swapping the order of these (the return type and the annotation). Was this intentional?

Reply:
__declspec(dllexport)/__declspec(dllimport) should be of left-side */& (MSVC specfic). This change for one-style with operator<<